python/mxnet/optimizer.py

```diff
@@ -40,10 +40,39 @@ class Optimizer(object):
 
     begin_num_update : int, optional
         The initial number of updates.
+
+    do_pruning : boolean, optional
```

Comment:
I have created a subclass, SparseSGD.
Want to be able to call SGD update() without calling _update_count(). Can that be done?